CMakeLists.txt

```diff
@@ -445,6 +445,7 @@ function(add_llgl_example_project PROJECT_NAME LINKER_LANG SRC_FILES LIB_FILES)
         ${PROJECT_NAME} PROPERTIES
             DEBUG_POSTFIX               "D"
             RUNTIME_OUTPUT_DIRECTORY    "${LLGL_OUTPUT_DIR}"
+            BUILD_RPATH                 "${LLGL_OUTPUT_DIR}"
```

Comment:
This should be `\$ORIGIN` instead of `${LLGL_OUTPUT_DIR}` so that you can move the executable and still load it from the directory where the program is currently located. This way you're fixing this old bandaid and still have the behavior unchanged.
I tested both solutions and this is what `objdump` shows when inspecting `libLLGL.so`:
```
$ objdump -x libLLGL.so |grep 'R.*PATH'
  RUNPATH              /home/lh/Development/Projects/LLGL/repo/build_linux/build:
$ objdump -x libLLGL.so |grep 'R.*PATH'
  RUNPATH              $ORIGIN:
```